src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -26,7 +26,7 @@ the MIT License. See LICENSE in the project root for license information. -->
     <ScrollViewer>
         <StackPanel Style="{StaticResource SettingsStackStyle}">
             <!--Theme-->
-            <ContentPresenter Style="{StaticResource SettingContainerStyle}">
+            <ContentPresenter>

```

Comment:
```suggestion
            <ContentPresenter Style={StaticResource SettingContainerStyle}"
                              Margin="0">
```
nit: could you keep the style, but just override the margin like this? That way, if we decide to add more stuff to `SettingContainerStyle`, we just get it for free.

Reply:
You do this in just about every page too, so please rinse-and-repeat.